commerce-api/src/main/java/com/liferay/commerce/discount/CommerceDiscountValue.java

```diff
@@ -46,7 +47,7 @@ public long getId() {
 	}
 
 	public BigDecimal[] getPercentages() {
-		return _percentages;
+		return _percentages.clone();
```

Comment:
use portal standard

Reply:
@ethanbustad So .clone() can be used on Arrays but instead of using ObjectUtils.clone() we use .clone() and cast the cloned object to the original type. Is this because ObjectUtils isn't a Liferay package? If it was used it would eliminate the need for casting.